src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -308,8 +308,8 @@ void get_calls_to_global_stmts(Allocator &al, const Location &loc, SymbolTable*
     }
 }
 
-template <class Struct>
-class CommonVisitor : public AST::BaseVisitor<Struct> {
+template <class StructType>
+class CommonVisitor : public AST::BaseVisitor<StructType> {
```

Comment:
```suggestion
template <class Struct>
class CommonVisitor : public AST::BaseVisitor<Struct> {
```
If above gets reverted, change this as well